api/client/api_client.go

```diff
@@ -0,0 +1,139 @@
+package client
```

Comment:
I'd recommend renaming the parent directory (and package name) to be `api_service_client`. That way, when people import it, it'll be very obvious in their code what's going on